src/uct/ib/base/ib_iface.c

```diff
@@ -479,6 +479,40 @@ static ucs_status_t uct_ib_iface_create_cq(uct_ib_iface_t *iface, int cq_length,
     return status;
 }
 
+
+static ucs_status_t uct_ib_iface_set_moderation(struct ibv_cq *cq,
+                                                const uct_ib_iface_config_t *config)
+{
+    ucs_status_t status = UCS_OK;
+
+#if HAVE_DECL_IBV_EXP_CQ_MODERATION
+    if (config->cq_moderation_count > UINT16_MAX) {
+        ucs_error("CQ moderation count is too high: %u", config->cq_moderation_count);
+        return UCS_ERR_INVALID_PARAM;
+    }
+
+    if (config->cq_moderation_period > UINT16_MAX) {
+        ucs_error("CQ moderation period is too high: %u", config->cq_moderation_period);
+        return UCS_ERR_INVALID_PARAM;
+    }
+
+    if (config->cq_moderation_count < UINT16_MAX ||
```

Comment:
code style, need to use () for each statement

Reply:
fixed